ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -0,0 +1,140 @@
+        super().__init__(driver)
+
+    @property
+    def select_mode(self):
+        return self.mode
+
+    @select_mode.setter
+    def select_mode(self, mode):
+        if not isinstance(mode, str):
+            raise ValueError("mode arg must be a string")
+        self.mode = mode
+
+    @property
+    def select_storage_class(self):
+        return self.storage_class_type
+
+    @select_storage_class.setter
+    def select_storage_class(self, storage_class):
+        if not isinstance(storage_class, str):
+            raise ValueError("storage class arg must be a string")
+        self.storage_class = storage_class
+
+    @property
+    def select_osd_size(self):
+        return self.osd_size
+
+    @select_osd_size.setter
+    def select_osd_size(self, osd_size):
+        if not isinstance(osd_size, str):
+            raise ValueError("osd size arg must be a string")
+        self.osd_size = osd_size
+
+    @property
+    def select_encryption(self):
+        return self.is_encryption
+
+    @select_encryption.setter
+    def select_encryption(self, is_encryption):
+        if not isinstance(is_encryption, bool):
+            raise ValueError("is_encryption arg must be a bool")
+        self.is_encryption = is_encryption
+
+    def install_ocs_opeartor(self):
+        """
+        Install OCS Opeartor
+
+        """
+        logger.info("Click On Operators Tab")
+        self.do_click(deployment["Operators Tab"])
+
+        logger.info("Click On OperatorHub Tab")
+        self.do_click(deployment["OperatorHub Tab"], type=By.LINK_TEXT)
+
+        logger.info("Search OCS Operator")
+        self.do_send_keys(
+            deployment["Search Operators"],
+            text="OpenShift Container Storage",
+            type=By.CSS_SELECTOR,
+        )
+
+        logger.info("Choose OCS")
+        self.do_click(deployment["Choose OCS"])
+
+        logger.info("Click Install OCS")
+        self.do_click(deployment["Click Install OCS"], type=By.CSS_SELECTOR)
+
+        logger.info("Click On Operators Tab")
+        self.do_click(deployment["Operators Tab"])
+
+    def install_storage_cluster(self):
+        """
+        Install Storage Cluster
+
+        """
+        logger.info("Click On Operators Tab")
+        self.do_click(deployment["Operators Tab"])
+
+        logger.info("Click On OperatorHub Tab")
+        self.do_click(deployment["Installed Operators Tab"], type=By.LINK_TEXT)
+
+        logger.info("Click On OCS Installed")
+        self.do_click(deployment["OCS Installed"], type=By.CSS_SELECTOR)
+
+        logger.info("Storage Cluster Tab")
+        self.do_click(deployment["Storage Cluster Tab"], type=By.CSS_SELECTOR)
+
+        logger.info("Click On 'Create Storage Cluster'")
+        self.do_click(deployment["Create Storage Cluster"], type=By.CSS_SELECTOR)
+
+        if self.mode == "internal":
+            self.install_internal_cluster()
+        elif self.mode == "lso":
+            self.install_lso_cluster()
+
```

Comment:
Need a verification function to verify from UI that storage cluster is installed.